src/core/namespace.c

```diff
@@ -3996,6 +3997,84 @@ static const char* const private_bpf_table[_PRIVATE_BPF_MAX] = {
 
 DEFINE_STRING_TABLE_LOOKUP(private_bpf, PrivateBPF);
 
+#include "bpf-delegate-configs.inc"
+
+DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_cmd, uint64_t);
+
+int bpf_dump_delegate_commands(FILE *fd, uint64_t u) {
```

Comment:
FILE is not a fd. Please rename to `f`.

BTW, I do not think it is necessary to introduce similar functions.
Please drop this and always use bpf_delegate_commands_to_string().